doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -1873,6 +1873,63 @@ The API call requires a Json body that includes the list of the fileIds that the
   export JSON='{"fileIds":[300,301]}'
 
   curl -H "X-Dataverse-key: $API_TOKEN" -H "Content-Type:application/json" "$SERVER_URL/api/datasets/:persistentId/files/actions/:unset-embargo?persistentId=$PERSISTENT_IDENTIFIER" -d "$JSON"
+  
+  
+Get the Archival Status of a Dataset By Version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Archiving is an optional feature that may be configured for a Dataverse instance. When enabled, this API call be used to retrieve the status. Note that this requires "superuser" credentials.
```

Comment:
Kind.of a quibble, but are we using "installation" as the term here or instance?

Reply:
FWIW: A quick scan through the guides shows some of each - installation is more popular. I see a dozen+ 'Dataverse instance' and lots of 'instance' referring to other components.

Reply:
FWIW, I prefer "Dataverse installation" to "Dataverse instance".